packages/cli/src/lib/core/prompt.ts

```diff
@@ -34,7 +34,7 @@ export function askToSkip(
   question: string,
   cliProcess: CLIProcess,
   options: {
-    precondition?: (r?: RcJson) => Promise<boolean>;
+    precondition?: (r: RcJson) => Promise<boolean>;
```

Comment:
I changed the type of the `askToSkip` predicate (precondition) but its only used there. However, when we decided to use it somewhere else, it should probably be re-written to not require positional and have a more abstract predicate type.